draft-poidt-teas-actn-poi-assurance.md

```diff
@@ -1,61 +1,195 @@
+~~~~
+{: #fig-ref-architecture title="Reference Network (copy of Figure 1 of RFC YYYY)"
+artwork-name="reference-architecture.txt"}
+
+EDITORS NOTE: Replace RFC YYYY with the RFC number of {{!I-D.ietf-teas-actn-poi-applicability}} once it has been published.
+
+In general, assurance involves detection and localization of link failures and performance degradation as well as re-routing (protection).
+
+Two cases will be considered:
+1. using grey interfaces on routers' ports, as outlined in {{!I-D.ietf-teas-actn-poi-applicability}}
+2. using colored optical interfaces on routers' ports, as outlined in {{?I-D.mix-teas-actn-poi-extension}}
+
+NOTE: It is not fully clear how much commonalities there are in service assurance for these two cases. This draft will start addressing both cases and assess at a later stage whether it is worthwhile keeping everything in a single draft or to split into two drafts.
+
+The MDSC is responsible for coordinating the whole multi-domain, multi-layer (packet and optical) network. MDSC interacts with different Provisioning Network Controllers (O/P-PNCs) through the MPI interface.
+The MPI interface presents an abstracted topology to MDSC, hiding the technology-specific aspects of the network and the topology details (depending on the policy chosen regarding the level of abstraction supported).
+
+Following the assumptions of section 2.1.2 of {{I-D.ietf-teas-actn-poi-applicability}}, this document analyses scenarios where
+the MDSC uses the partial summarization approach to coordinate multi-domain/multi-layer path computation.
+
+In this approach, the MDSC has complete visibility of the TE topology of the packet network domains and an abstracted
+view of the TE topology of the optical network domains.
+That means the MDSC has the capability of performing multi-domain/single-layer path computation for the packet layer. The MDSC needs to delegate the O-PNCs to perform local path computation within their respective domains.
+It uses the information received by the O-PNCs and its TE topology view of the multi-domain packet layer to perform multi-layer/multi-domain path computation.
+
+P-PNCs are responsible for setting up the TE paths between any two PEs or BRs in their respective controlled domains,
+as requested by MDSC, and providing topology information to the MDSC.
+
+O-PNCs are responsible to provide to the MDSC an abstract TE topology view of their underlying optical network resources.
+They perform single-domain local path computation, when requested by the MDSC. They also perform optical tunnel setup, when requested by the MDSC.
+
+No GMPLS-UNI interaction between IP and Optical equipment is considered.
+This is also the assumption followed in this document: the MDSC performs the function of multi-layer/multi-domain path computation
+through the same mechanisms described in (I-D.ietf-teas-actn-poi-applicability) even if that is applied to the maintenance and protection cases described in the next sections.
```

Comment:
not sure I understand the sentence "even if that is applied to the maintenance and protection cases"

Reply:
Right, that sentence does add any informative value. I just wanted to say that we are "extending" the analysis to the cases described by Fabio. I will remove it from the main text.